app/services/platforms/index.ts

```diff
@@ -255,9 +258,12 @@ export function getPlatformService(platform: TPlatform): IPlatformService {
     facebook: FacebookService.instance,
     tiktok: TiktokService.instance,
     trovo: TrovoService.instance,
+    twitter: TwitterPlatformService.instance,
   }[platform];
 }
 
+window['getPlatformService'] = getPlatformService;
```

Comment:
Might be a debugging left-over, but don't mind me if intended.

Reply:
yup debugging - ty!